app/controllers/to_do_items_controller.rb

```diff
@@ -26,6 +26,11 @@ def move
     render nothing: true
   end
 
+  def statistic
+    digest = DailyDigest.new
+    digest.send
+  end
+
```

Comment:
@vlasiak, Looks :+1: Also, Ruby developers like to do in one line:
```ruby
DailyDigest.new.perform
```
I suggest to use `perform`, it's like a convention (aka hook) for the objects that just do one thing.
This is just a heads up.